From 6721240d0de7c631e1ba3c8f59ebec324106ded4 Mon Sep 17 00:00:00 2001
From: Zhu Yanjun <yanjun.zhu@windriver.com>
Date: Thu, 2 Jul 2015 15:59:29 +0800
Subject: [PATCH 1/2] igb: enable DISABLE_PACKET_SPLIT option

In igb, the option DISABLE_PACKET_SPLIT is not enabled. AH
packets are modified in igb driver. As such, these AH packets
can not pass ICV.

Suggested-by: Bruce Ashfield <bruce.ashfield@windriver.com>
Signed-off-by: Zhu Yanjun <yanjun.zhu@windriver.com>
---
 drivers/net/igb/Makefile |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/net/igb/Makefile b/drivers/net/igb/Makefile
index 1aebda7..9a814f6 100644
--- a/drivers/net/igb/Makefile
+++ b/drivers/net/igb/Makefile
@@ -36,6 +36,8 @@ igb-objs := igb_main.o  e1000_82575.o e1000_mac.o e1000_phy.o \
 	    kcompat.o e1000_api.o e1000_mbx.o e1000_i210.o \
 	    igb_vmdq.o igb_procfs.o igb_hwmon.o igb_debugfs.o
 
+EXTRA_CFLAGS += -DDISABLE_PACKET_SPLIT 
+
 ifeq ($(ARCH),alpha)
   EXTRA_CFLAGS += -ffixed-8 -mno-fp-regs
 endif
-- 
1.7.5.4

